art/api/tests/test_asset_category_model.py

```diff
@@ -0,0 +1,27 @@
+from django.test import TestCase
+from ..models import AssetCategory
+
+
+class AssetCategoryModelTest(TestCase):
+    """ Tests for the Asset Category Model """
+    def setUp(self):
+        AssetCategory.objects.create(name="Accessories")
+        self.category = AssetCategory.objects.get(name="Accessories")
+
+    def test_can_save_a_category(self):
+        self.assertIn("Accessories", self.category.name)
```

Comment:
we might need to create a new category here and check for an increase in the category count.

Reply:
Alright makes sense! I have changed this.